askama_derive/src/generator.rs

```diff
@@ -241,6 +258,63 @@ fn find_used_templates(
     Ok(())
 }
 
+#[derive(Default)]
```

Comment:
This type should go towards the bottom, just above the `Writable` type definition. I think we should ditch pretty much all the methods and trait impls for it other than the `push()` method, in favor of direct access to its fields.